frontend/src/widgets/DashboardOverviewContainer.scss

```diff
@@ -0,0 +1,12 @@
+@media( min-width: 640px){
+    .smart-hub--dashboard-overview-container {
+        column-gap: 1rem;
+        row-gap:.5rem;
+    }
+}
+
+@media( min-width: 1024px ){
+    .smart-hub--dashboard-overview-container {
+        gap: 1em;
+    }
+}
```

Comment:
Suggestion:

1) Delete this file.
2) Remove "margin-bottom-1" from the classnames OverviewWidgetField line 29 
3) add "flex-gap-2" to the classnames in "DashboardOverviewContainer" line 32

Reply:
changes made.